mwdb/core/plugins.py

```diff
@@ -78,6 +78,18 @@ def on_created_text_blob(self, blob: TextBlob):
     def on_reuploaded_text_blob(self, blob: TextBlob):
         pass
 
+    @hook_handler_method
+    def on_created_tag(self, object: Object, tag: Tag):
+        pass
+
+    @hook_handler_method
+    def on_reuploaded_tag(self, tag: Tag):
```

Comment:
There is missing argument (based on `on_created_tag` signature and caller function)
```suggestion
    def on_reuploaded_tag(self, object: Object, tag: Tag):
```